Riot/Modules/Room/BubbleReactions/BubbleReactionsView.swift

```diff
@@ -0,0 +1,136 @@
+/*
+ Copyright 2019 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import MatrixSDK
+import Reusable
+
+@objcMembers
+final class BubbleReactionsView: UIView, NibOwnerLoadable {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let minimumInteritemSpacing: CGFloat = 6.0
+        static let minimumLineSpacing: CGFloat = 2.0
+    }
+    
+    // MARK: - Properties
+    
+    // MARK: Outlets
+
+    @IBOutlet private weak var collectionView: UICollectionView!
+    
+    // MARK: Private
+    
+    private var reactionsViewData: [BubbleReactionViewData] = []
+    private var theme: Theme?
+    
+    // MARK: Public
+    
+    // Do not use `BubbleReactionsViewModelType` here due to Objective-C incompatibily
+    var viewModel: BubbleReactionsViewModel? {
+        didSet {
+            self.viewModel?.viewDelegate = self
+            self.viewModel?.process(viewAction: .loadData)
+        }
+    }
+    
+    // MARK: - Setup
+    
+    private func commonInit() {
+        self.collectionView.isScrollEnabled = false
+        self.collectionView.delegate = self
+        self.collectionView.dataSource = self
+        
+        if let collectionViewFlowLayout = self.collectionView.collectionViewLayout as? UICollectionViewFlowLayout {
```

Comment:
I wonder if we should use `LeftAlignedCollectionViewFlowLayout` defined at https://stackoverflow.com/a/36016798 to have the correct left alignement.

And then set it like:
`self.collectionView.setCollectionViewLayout(LeftAlignedCollectionViewFlowLayout(), animated: false)`

Reply:
Yes he should use a custom `UICollectionViewLayout` to handle one item case